tests/test-extensions/heisenberg-extension/src/main/java/org/mule/test/heisenberg/extension/HeisenbergOperations.java

```diff
@@ -272,9 +273,15 @@ public void die(@Config HeisenbergExtension config) {
     return enemies;
   }
 
+  @Deprecated(message = "The usage of this operation must be replaced by the knock operation.", since = "1.5.0",
+      removedIn = "2.0.0")
```

Comment:
is not correct the name `removedIn`, it tells as a past action that has been already removed, should be change to something like `toRemoveIn` or something like that